api/src/test/java/org/pmiops/workbench/leonardo/LeonardoLabelHelperTest.java

```diff
@@ -0,0 +1,25 @@
+package org.pmiops.workbench.leonardo;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+public class LeonardoLabelHelperTest {
```

Comment:
Should `appTypeToLabelValue` also be tested? Or do you consider it tested by the API client tests?

Reply:
added a new tests. PTAL
Original that was covered in LeoApiTest, and I feel that is too simple to add test. 
But couldn't hurt to add simple test